Cargo.toml

```diff
@@ -37,6 +37,8 @@ resolver = "2"
 [patch.crates-io]
 fluvio-future = { git = "https://github.com/infinyon/future-aio.git", branch = "master"}
 #fluvio-future = { path = "../future-aio" }
+#k8-types = { path = "../k8-api/src/k8-types" }
+k8-types = { git = "https://github.com/tjtelan/k8-api.git", branch = "add-node" }
```

Comment:
should use release version?

Reply:
Yeah, we should. I have a PR open and can make this change soon. https://github.com/infinyon/k8-api/pull/51